src/features/cartography/infrastructure/configuration/tokens/marker/markers.token.ts

```diff
@@ -1,50 +1,78 @@
-const CNFS_MARKER_CLUSTER_DIMENSIONS: Point = new Point(
+const CNFS_MARKER_CLUSTER_DIMENSIONS: LeafletPoint = new LeafletPoint(
   CNFS_MARKER_CLUSTER_WIDTH_IN_PIXEL,
   CNFS_MARKER_CLUSTER_HEIGTH_IN_PIXEL,
   ROUND_FALSE
 );
 
 const USAGER_MARKER_WIDTH_IN_PIXEL: number = 16;
 const USAGER_MARKER_HEIGTH_IN_PIXEL: number = 16;
-const USAGER_MARKER_DIMENSIONS: Point = new Point(USAGER_MARKER_WIDTH_IN_PIXEL, USAGER_MARKER_HEIGTH_IN_PIXEL, ROUND_FALSE);
+const USAGER_MARKER_DIMENSIONS: LeafletPoint = new LeafletPoint(
+  USAGER_MARKER_WIDTH_IN_PIXEL,
+  USAGER_MARKER_HEIGTH_IN_PIXEL,
+  ROUND_FALSE
+);
 
-export enum AvailableMarkers {
+export enum Marker {
   Cnfs = 'cnfs',
   CnfsCluster = 'cnfsCluster',
   Usager = 'usager'
 }
 
-export const MARKERS: Record<AvailableMarkers, MarkerConfiguration> = {
-  [AvailableMarkers.Cnfs]: {
-    iconAnchor: new Point(CNFS_MARKER_DIMENSIONS.x * HALF, CNFS_MARKER_DIMENSIONS.y),
-    iconSize: CNFS_MARKER_DIMENSIONS,
-    iconUrl: 'assets/map/pin-cnfs.svg'
-  },
-  [AvailableMarkers.CnfsCluster]: {
-    iconAnchor: new Point(CNFS_MARKER_CLUSTER_DIMENSIONS.x * HALF, CNFS_MARKER_CLUSTER_DIMENSIONS.y),
-    iconSize: CNFS_MARKER_CLUSTER_DIMENSIONS,
-    iconUrl: 'assets/map/pin-cnfs-cluster.svg'
-  },
-  [AvailableMarkers.Usager]: {
-    iconAnchor: new Point(USAGER_MARKER_DIMENSIONS.x * HALF, USAGER_MARKER_DIMENSIONS.y),
-    iconSize: USAGER_MARKER_DIMENSIONS,
-    iconUrl: 'assets/map/pin-usager.svg'
-  }
+export const MARKERS: Record<Marker, (feature?: Feature<Point, MarkerProperties>) => DivIcon | Icon> = {
+  [Marker.Cnfs]: (): Icon =>
+    icon({
+      iconAnchor: new LeafletPoint(CNFS_MARKER_DIMENSIONS.x * HALF, CNFS_MARKER_DIMENSIONS.y),
+      iconSize: CNFS_MARKER_DIMENSIONS,
+      iconUrl: 'assets/map/pin-cnfs.svg'
+    }),
+  // eslint-disable-next-line max-lines-per-function
+  [Marker.CnfsCluster]: (feature?: Feature<Point, MarkerProperties>): DivIcon =>
+    new DivIcon({
+      className: '',
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      html: `<div
+        style="width: 31px;
+               height: 36px;
+               font-family: Marianne, sans-serif;
+               font-weight: 600;
+               color: #ffff;
+               text-align: center;
+               background-image: url('./assets/map/pin-cnfs-cluster.svg');
+               background-repeat: no-repeat;
+               background-position: center;
+               background-size: contain;">
+      ${feature?.properties['point_count_abbreviated'] as number}</div>`,
+      iconAnchor: new LeafletPoint(CNFS_MARKER_CLUSTER_DIMENSIONS.x * HALF, CNFS_MARKER_CLUSTER_DIMENSIONS.y),
+      iconSize: CNFS_MARKER_CLUSTER_DIMENSIONS
+    }),
+  [Marker.Usager]: (): Icon =>
+    icon({
+      iconAnchor: new LeafletPoint(USAGER_MARKER_DIMENSIONS.x * HALF, USAGER_MARKER_DIMENSIONS.y),
+      iconSize: USAGER_MARKER_DIMENSIONS,
+      iconUrl: 'assets/map/pin-usager.svg'
+    })
```

Comment:
```suggestion
    [Marker.Cnfs]: cnfsMarkerFactory,
    [Marker.CnfsCluster]: cnfsClusterMarkerFactory,
    [Marker.Usager]: usagerMarkerFactory
```
Avec extraction des factories dans des fonctions dédiées plutôt que des fonctions annonymes => à mettre dans un autre fichier, par exemple `markers.factories.ts`